docs/server_config.md

```diff
@@ -62,24 +62,52 @@ Note:
 
 ## Timeplus Appserver configurations {#appserver}
 
+Timeplus Appserver read configurations from configuration file (.yaml) and environment variables. The default values listed below are the default values that hardcoded inside the binary file. The actual default value may be different depends on your deployement (bare-metal or k8s via helm)
```

Comment:
read->reads

Reply:
`the` configuration file

Reply:
that `are` hardcoded

Reply:
`k8s via helm` -> `Kubernetes`
k8s is okay to be part of the URL but in general we should not use this in the doc content.

Reply:
thanks! updated